components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/tests/Controller/DmApiPublisherMethodControllerTests.cs

```diff
@@ -81,7 +81,7 @@ public class DmApiPublisherControllerTests {
 
                 publishNodesResult.Subject.StatusMessage.First()
                     .Should()
-                    .Be("Succeeded");
+                    .Contain("succeeded");
```

Comment:
is this test still successful? since we return an empty list?

Reply:
The list would not be empty so this test passes.